packages/sirius-web/frontend/sirius-web/src/index.tsx

```diff
@@ -34,20 +38,28 @@ if (process.env.NODE_ENV !== 'production') {
   loadErrorMessages();
 }
 
+const registry = new ExtensionRegistry();
+
+const apolloClientOptionsConfigurer: ApolloClientOptionsConfigurer = (currentOptions) => {
+  return {
+    ...currentOptions,
+    documentTransform: ellipseNodeStyleDocumentTransform,
```

Comment:
If another configurer has created some documentTransform, it will override it, we should consider merging them instead

Reply:
Nobody does that today but the code of `sirius-web-application` may change later

Reply:
Yes I had the `concat `method before when I created a dedicated extension point. I removed it because I then thought that it would be the responsibility of the contributor then.